src/app/App.js

```diff
@@ -61,6 +61,14 @@ class App extends EventEmitter {
 		this.captureScrollPositionFromScrollEvent = true;
 
 		/**
+		 * Holds the value of the current browser path.
+		 * @type {!string}
+		 * @default the current browser path.
+		 * @protected
+		 */
+		this.currentBrowserPath = window.location.pathname;
```

Comment:
The suggestion was to not store the value and instead read from the current browser path - from `utils` or `location`. It's confusing to have a property that stores the browser current url since it's already available for you from the api. Maybe its just a naming thing, what you need here is not the `currentBrowserPath` is a `browserPathBeforeSomethingToHappen`. That make sense?